requirements-dev.txt

```diff
@@ -1,8 +1,245 @@
-boto3>=1.15
```

Comment:
Same as requirements.txt, moved to pyproject.toml. This is not generated from that with `poetry export --dev`.